tuf/tuf.go

```diff
@@ -165,15 +184,14 @@ func (tr *Repo) RemoveBaseKeys(role string, keyIDs ...string) error {
 // will be added in front of (i.e. higher priority) in the delegation list.
 // An empty before string indicates to add the role to the end of the
 // delegation list.
-// A new, empty, targets file will be created for the new role.
 func (tr *Repo) UpdateDelegations(role *data.Role, keys []data.PublicKey, before string) error {
```

Comment:
No. It's mean to be a way to support the TUF specs idea about order of
roles indicating priority. I've yet to climb down off the fence on whether
I'm going to implement it or not (because I don't think we're going to use
it) so I keep leaving the argument there... :-/